pkg/apiserver/handlers_test.go

```diff
@@ -299,3 +299,31 @@ func TestGetAPIRequestInfo(t *testing.T) {
 		}
 	}
 }
+
+func TestIsAPIRequest(t *testing.T) {
+	tests := map[string]struct {
+		url      string
+		expected bool
+	}{
+		"simple groupless": {"/api/version/resource", true},
```

Comment:
Add tests for following cases, or something close to them:
 `apis/extensions/v1beta1/job`
 `apis/extensions/v1beta1/job/foo`
 `apis/extensions/v1beta1/job/foo/scale`